utilities/getArticleByTitle/index.ts

```diff
@@ -0,0 +1,29 @@
+import { APPLICATION_JSON, CONTENTFUL_API_ACCESS_TOKEN, CONTENTFUL_API_URL, POST } from '../constants'
+import { ArticleProps } from '../../components/Article'
+
+export default async function getArticleByTitle(title: string): Promise<ArticleProps> {
+  const GET_ARTICLE_BY_TITLE_GRAPHQL_QUERY = `
+    {
+      articleCollection(where: {title: "${title}"}) {
+        items {
+          title
+          content
+        }
+      }
+    }
+  `
```

Comment:
Should the collection be called `pages`, `pageContent`, or something else? What do you think? Remember to add `slug` and make it unique.

Reply:
I think the response will have only one page without a limitation. So pageContent sounds good to me.